app/services/imports/lettings_logs_import_service.rb

```diff
@@ -226,7 +226,7 @@ def create_log(xml_doc)
       # Sets the log creator
       owner_id = field_value(xml_doc, "meta", "owner-user-id").strip
       if owner_id.present?
-        attributes["created_by"] = User.find_by(old_user_id: owner_id)
+        attributes["created_by"] = LegacyUser.find_by(old_user_id: owner_id)&.user
```

Comment:
Given that a missing `created_by` attribute will cause issues I wonder whether logging something here already will help speed up investigating.

Reply:
Sure, see: [7aacbbc](https://github.com/communitiesuk/submit-social-housing-lettings-and-sales-data/pull/894/commits/7aacbbc624a3b4634110d2662b0a56466ad4c69b)